src/components/app/App.tsx

```diff
@@ -1,22 +1,25 @@
 import React from 'react';
-import logo from '../../logo.svg';
 import './App.css';
 import Nav from '../nav/Nav';
 import Home from '../home/Home';
 import Teas from '../teas/Teas';
-import { Routes, Route } from 'react-router-dom';
-// import { useState, useEffect } from 'react';
+import Error from '../error/Error'
+import { Routes, Route} from 'react-router-dom';
+import { useState} from 'react';
 
 function App() {
 
+  const [isRedirected, setIsRedirected] = useState<boolean>()
+  console.log('app state isRedirected', isRedirected)
```

Comment:
Let's remove this once we get the rerouting working!

Reply:
100% agree!